main.go

```diff
@@ -227,13 +227,13 @@ func muteDetector(detectorID string, silence time.Duration) error {
 	}
 	defer resp.Body.Close()
 
-	if resp.StatusCode != 200 {
+	if resp.StatusCode != 201 {
```

Comment:
does it always return 201, or sometimes 200 and sometimes 201?

Reply:
Since it's a POST (rather than PUT) I'm guessing always a 201 (created) but I can extend it to check for both, if you'd prefer.

Reply:
The docs don't really indicate either way -- which is how I missed the 201 initially.